pkgs/applications/accessibility/nerd-dictation/default.nix

```diff
@@ -0,0 +1,51 @@
+{ fetchFromGitHub
+, substituteAll
+, lib
+, stdenv
+, python3
+, pulseaudio
+, xdotool
+, ydotool
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "nerd-dictation";
+  version = "unstable-2022-07-12";
+
+  src = fetchFromGitHub {
+    owner = "ideasman42";
+    repo = "nerd-dictation";
+    rev = "6194641846e55b89c955a0fd6b3ac83bfd52b7d7";
+    hash = "sha256-fvdh2eZHh4Ups85BNejedo/tSf3cb/C7pJvgf9zKqoU=";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./fix-paths.patch;
+      parec = "${pulseaudio}/bin/parec";
+      xdotool = "${xdotool}/bin/xdotool";
+      ydotool = "${ydotool}/bin/ydotool";
+    })
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    vosk-python
+  ];
+
+  postPatch = ''
+    cd package/python
+  '';
+
+  # No tests.
+  doCheck = false;
```

Comment:
pythonImportsCheck missing

Reply:
This is an app not a library so not sure what would we even import.